applications/vanilla/models/class.discussionmodel.php

```diff
@@ -857,6 +859,17 @@ public function getWhere($where = false, $orderFields = '', $orderDirection = ''
                 ->select('w.Participated');
         }
 
+        // Add select of addition fields to the SQL object.
+        if (!empty($selects)) {
+            if (!is_array($selects)) {
+                $selects = [$selects];
+            }
+
```

Comment:
Why do we need to check type?
We just introduced new variable and defined initial value as empty array.
I feel pretty safe to run `foreach` without extra checks.
```suggestion
```

Reply:
I see your point. I saw this type of check somewhere else in the code and I assumed it would be flexible to allow assigning a value directly. I can remove it if it doesn't bring any value.